src/main.rs

```diff
@@ -26,14 +27,21 @@ fn main() {
     }
     println!("✅ Permission granted");
 
-    // #3 Get recording targets (WIP)
+    // #3 Get recording targets
     let targets = scap::get_all_targets();
     println!("🎯 Targets: {:?}", targets);
 
+    let target = targets.into_iter().find(|target| match target {
+        Target::Display(_) => false,
+        Target::Window(w) => w.title.contains("Visual Studio Code"),
+    });
+
+    println!("{:?}", target);
+
```

Comment:
At the moment will this work for Mac?
What happens if there exists no window with `Visual Studio Code` in the title?

Also, let's remove the `println!` from here

Reply:
Targets aren't implemented on macOS yet, I will pick it up after I get an update on this: https://github.com/helmerapp/scap/pull/72#issuecomment-2136017342